### ISSUE TITLE
syntactic parsing

### ISSUE THREAD

**Issue by nelson-liu** [nlp-compromise/nlp_compromise#110]
Hi,
In the readme, you state that you get a "86% on the Penn treebank". Are you doing parsing in this library? If so, I wasn't able to find any docs on it, could you perhaps guide me to the correct module. If not, what do you mean when you say "86% on the Penn treebank"?

Thanks!

**Comment by spencermountain** [comment]
hi nelson! yeah, i haven't done a good job at keeping these organised. for the penn version, i used 
[this file (big)](https://raw.githubusercontent.com/nlp-compromise/nlp_compromise/2e657cd4564fc25b5e271d008c1032610e586a33/test/perf_tests/pos_test/pennTreebank.js) and mapped its tagset to mine, which got messier and messier. Importantly, I ignored sentences that had different # of terms, and only gave a yes/no if the sentence proved to be a perfect match.

This was about a year ago. Since I added phrasal-verb chunking, and a bunch of adhoc lumping-term rules, it became harder and harder to compare the two datasets. 

i've kept the other performance tests in `./tests/perf_tests`, if you wanna run them. I should probably figure-out a way to properly evaluate POS again, now that it's strayed from the Penn output considerably.

i do some sanity test stuff on a wikipedia corpus in `./tests/perf_tests/pos`, but it doesn't give a % correct result, only which methods are heating up, etc.
cheers
